requirements/edx/base.txt

```diff
@@ -129,7 +129,6 @@ help-tokens==1.1.2        # via -r requirements/edx/base.in
 html5lib==1.1             # via -r requirements/edx/base.in, ora2
 icalendar==4.0.7          # via -r requirements/edx/base.in
 idna==2.10                # via -r requirements/edx/paver.txt, requests
-importlib-metadata==1.7.0  # via -c requirements/edx/../constraints.txt, -r requirements/edx/paver.txt, kombu, path
```

Comment:
Not sure why its removing from here.

Reply:
I'm guessing you ran `make upgrade` under Python 3.8, whereas Jenkins is still using 3.5 since we haven't turned off testing against 3.5 yet.

Reply:
ah, yes we ran this on our make upgrade machine.